README.rst

```diff
@@ -66,6 +42,26 @@ Why use coala?
   without you having to worry.
 - *Unified interface*: One tool for all programming languages.
```

Comment:
Also at Line 56, I think it should be `Read more in our documentation`, instead of `Read more at our documentation`.